avm/res/document-db/database-account/sql-database/main.bicep

```diff
@@ -0,0 +1,69 @@
+metadata name = 'DocumentDB Database Account SQL Databases'
+metadata description = 'This module deploys a SQL Database in a CosmosDB Account.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Conditional. The name of the parent Database Account. Required if the template is used in a standalone deployment.')
+param databaseAccountName string
+
+@description('Required. Name of the SQL database .')
+param name string
+
+@description('Optional. Array of containers to deploy in the SQL database.')
+param containers array = []
+
+@description('Optional. Request units per second. Will be set to null if autoscaleSettingsMaxThroughput is used.')
+param throughput int = 400
+
+@description('Optional. Specifies the Autoscale settings and represents maximum throughput, the resource can scale up to.  The autoscale throughput should have valid throughput values between 1000 and 1000000 inclusive in increments of 1000. If value is set to -1, then the property will be set to null and autoscale will be disabled.')
+param autoscaleSettingsMaxThroughput int = -1
```

Comment:
Please make nullable

Reply:
implemented in b9e0bc4e17a4442fab0428d88e20b7361fa3a305